install/dataloggers.csv

```diff
@@ -993,6 +993,7 @@ Nanometrics,Centaur CTR4-6S,9889,Paritu Road,,2023-06-26T20:28:00Z,9999-01-01T00
 Nanometrics,Centaur CTR4-6S,9891,Cape Kidnappers,,2023-05-25T04:20:00Z,9999-01-01T00:00:00Z
 Nanometrics,Centaur CTR4-6S,9893,Porangahau,,2023-05-04T02:09:00Z,9999-01-01T00:00:00Z
 Nanometrics,Centaur CTR4-6S,9894,"Amberley, Maskells Rd",,2023-06-27T00:27:00Z,9999-01-01T00:00:00Z
+Nanometrics,Centaur CTR4-6S,9895,Kahuranaki,,2023-07-19T23:31:00Z,9999-01-01T00:00:00Z
```

Comment:
Opening time is 2023-07-19T**21**:31:00Z?

Reply:
My bad, fixed now.